packages/cli/src/commands/multisig/transfer.ts

```diff
@@ -0,0 +1,48 @@
+import { flags } from '@oclif/command'
+import { BigNumber } from 'bignumber.js'
+import { BaseCommand } from '../../base'
+import { displaySendTx } from '../../utils/cli'
+import { Args, Flags } from '../../utils/command'
+
+export default class MultiSigTransfer extends BaseCommand {
+  static description =
+    'Ability to approve CELO transfers to and from multisig. Submit transaction or approve a matching existing transaction'
+
+  static flags = {
+    ...BaseCommand.flags,
+    to: Flags.address({ required: true, description: 'Recipient of transfer' }),
+    amount: flags.string({ required: true, description: 'Amount to transfer, e.g. 10e18' }),
+    transferFrom: flags.boolean({ description: 'Perform transferFrom instead of transfer' }),
+    sender: Flags.address({ description: 'Identify sender if performing transferFrom' }),
```

Comment:
I think you can specify that dependency in the flag itself with `dependsOn`

Reply:
✅